tests/integration/test_metadata_connection.py

```diff
@@ -1,12 +1,15 @@
 import pytest
 import em27_metadata
+
 import src
+from src.utils.envutils import get_config_path, config_dir_key
 
 
 @pytest.mark.order(3)
 @pytest.mark.integration
 def test_metadata_connection() -> None:
-    config = src.types.Config.load()
+    config_path = get_config_path(config_dir_key())
+    config = src.types.Config.load(config_path)
```

Comment:
I thought about this. `Config.load` has a named argument `path` with the default value which is the default config path. I suppose this is there so that the path can be changed to `config/config.template.json` when needed (in the tests?, see comment about the docs). We could check for the existence of the environment variable inside `load()`, but if there was an argument given for `path`, we would need to deal with that. That's why I used this existing `path` argument to pass the alternate path.